chia/full_node/full_node.py

```diff
@@ -221,37 +219,28 @@ async def _start(self) -> None:
         # These many respond_transaction tasks can be active at any point in time
         self.respond_transaction_semaphore = asyncio.Semaphore(200)
         # create the store (db) and full node instance
-        db_connection = await aiosqlite.connect(self.db_path)
-        db_version: int = await lookup_db_version(db_connection)
+        # TODO: is this standardized and thus able to be handled by DBWrapper2?
```

Comment:
to answer this question: the DB lookup could be handled by `DBWrapper2`, but the version number that's read from the DB is required to construct the DBWrapper object. So, we could create a temporary DBWrapper with version 1, read the version and the create the final DBWrapper with the correct version.

This is because storing the DB version in the DBWrapper object is quite convenient

Reply:
But if the wrapper handled this, maybe only in the `.create()` method, then it could fill it itself 'during creation'.

Reply:
I guess there may be a need for 'default' in case of initial creation.

Reply:
> But if the wrapper handled this, maybe only in the `.create()` method, then it could fill it itself 'during creation'.

only for the full-node's blockchain database. We use the `DBWrapper2` for other databases also, where we don't use the version number (yet)

Reply:
Could be optional.